css/interface.css

```diff
@@ -53,6 +53,15 @@
   border-bottom-color: #000201;
 }
 
+.btn:focus,
```

Comment:
Is this worth going into `fliplet-studio-ui`?

Reply:
That's a good question. I think so.
I'll update it and remove it from the component's code. Will do the same on https://github.com/Fliplet/fliplet-widget-app-submission/pull/138

Reply:
@tonytlwu removed the styles and added to `fliplet-studio-ui`. Ready to review.